api/IntersectionObserver.json

```diff
@@ -0,0 +1,52 @@
+{
+  "version": "1.0.0",
+  "data": {
+    "api": {
+      "IntersectionObserver": {
+        "__compat": {
+          "basic_support": {
+            "support": {
+              "chrome": {
+                "version_added": "51"
+              },
+              "edge": {
+                "version_added": true
+              },
+              "firefox": [
+                {
+                  "version_added": true,
+                  "version_removed": "55",
+                  "flag": {
+                    "type": "preference",
+                    "name": "dom.IntersectionObserver.enabled",
+                    "value_to_set": "true"
+                  },
+                  "notes": "Enabled by default in Firefox 55; see bug 1243846 for details."
+                },
+                {
+                  "version_added": "53",
+                  "notes": "Available but disabled by default in Firefox 53."
+                }
+              ],
+              "ie": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": "51.0"
```

Comment:
"51" and not "51.0" for `webview_android` and for `chrome_android` here and in the other file.